### DISCUSSION TITLE
Edit individual occurrences

### DISCUSSION THREAD

**Discussion by danieltott** [solspace/craft-calendar#110] [Feature Requests]
It would be really great to be able to edit individual occurrences of repeating events, similar to how iCal/google cal work. Currently, we make our users delete the occurrence and create a one-off event, but it's pretty hard to explain, plus the user needs to ensure they recreate everything properly.

**Comment by theyve** [comment]
Our clients would also heavily benefit from being able to individually edit the start and end times of single occurrences.
